apps/web/src/hooks/useApproveCallback.ts

```diff
@@ -131,8 +132,15 @@ export function useApproveCallbackFromTrade(
     () => (trade ? computeSlippageAdjustedAmounts(trade, allowedSlippage)[Field.INPUT] : undefined),
     [trade, allowedSlippage],
   )
-
-  return useApproveCallback(amountToApprove, ROUTER_ADDRESS[chainId])
+  const isAkkaSwap = useIsAkkaSwap()
+  return useApproveCallback(
```

Comment:
this needs fititng for the new ROUTER_ADDRESS structure (not indexing by integer index). Also please don't hardcode the ChainId's but use the ChainId.Bitgert.
Would also be great to not have this hardcoded here, but maybe have a list of supported chains for AKKA routing that simply sets isAkkaSwap to false if the chain is not supported.